apis/cpi/v1alpha1/vspherecpiconfig_types.go

```diff
@@ -161,6 +161,7 @@ type ParavirtualConfig struct {
 
 	// Used in vsphereParavirtual mode, defines the Cluster kind.
 	// +kubebuilder:validation:Optional
+	// +kubebuilder:default:=Cluster
 	ClusterKind string `json:"clusterKind"`
```

Comment:
I am kinda confused why this is being exposed in the API.

Reply:
there might be other type of Cluster CR such as `TanzuKubernetesCluster`. @lubronzhan could you confirm we still need this field?

Reply:
This field will be removed in a follow-up PR. It will derive other fields for para-virt mode as well.

Reply:
(future) Please log follow up issues and link them